prisma/schema.prisma

```diff
@@ -92,6 +101,52 @@ model Event {
   updatedAt DateTime @updatedAt
 }
 
+model Donor {
+  id                 Int            @id @default(autoincrement())
+  userId             Int?
+  email              String?
+  identification     String?
+  identificationType String?
+  cardStart          String?
+  paymentMethod      String?
+  donations          Donation[]
+  subscriptions      Subscription[]
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  @@unique([identification, identificationType])
+}
+
+model Donation {
+  id             Int     @id @default(autoincrement())
+  projectId      Int
+  project        Project @relation(fields: [projectId], references: [id])
+  donorId        Int
+  donor          Donor   @relation(fields: [donorId], references: [id])
+  amount         Int
+  mpPreferenceId String
+
+  createdAt DateTime @default(now())
+}
+
+model Subscription {
+  id                Int               @id @default(autoincrement())
+  projectId         Int
+  project           Project           @relation(fields: [projectId], references: [id])
+  donorId           Int
+  donor             Donor             @relation(fields: [donorId], references: [id])
+  amount            Int
+  frequency         Int
```

Comment:
frequency is how many times in the frequency interval. We will hardcode it to 1 every month. but it could then be implemented to do like 2 (freuqency) every year (frequency interval)